docs/_tutorials/ja_getting_started.md

```diff
@@ -152,7 +150,10 @@ ngrok http 3000
 **Request URL**  ボックスの  **Enable Events**  スイッチの下のフィールドにこの URL を貼り付けます。Bolt アプリが引き続き実行されている場合は、URL が検証されチェックマークが表示されます。
 
 Request URL が検証されたら、**Subscribe to Bot Events** までスクロールします。メッセージに関するイベントが４つあります−
```

Comment:
Not sure is the dash `-` at the end is appropriate. In the English docs it is a colon `:`

Reply:
Good catch!